packages/transforms/README.md

```diff
@@ -1 +1,55 @@
+# @endo/transforms
 
+> Source transforms for [SES](https://github.com/endojs/endo/tree/master/packages/ses)-enabled applications
+
+This package currently provides a transform to modify comments which would otherwise be rejected outright by SES.
+
+## Example
+
+```js
+// ESM example
+import { transformSource } from '@endo/transforms';
+import fs from 'node:fs/promises';
+
+const source = await fs.readFile('./dist/index.js', 'utf8');
+const sourceMap = await fs.readFile('./dist/index.js.map', 'utf8');
+const sourceUrl = 'index.js'; // assuming the source map references index.js
+const sourceType = 'script';
+
+const { code, map } = await transformSource(source, {
+  sourceMap,
+  sourceUrl,
+  sourceType,
+});
+
+await fs.writeFile('./dist/index.ses.js', code);
+await fs.writeFile('./dist/index.ses.js.map', JSON.stringify(map));
+```
+
+The censorship-evasion function can also be used directly, if you have parsed
+your source into an AST via Babel:
+
+```ts
+// TypeScript example
+import { evadeCensor } from '@endo/transforms';
+import { type Comment } from '@babel/types';
+
+/**
+ * This comment could be e.g., `@property {import('foo').Bar} bar`, which SES 
+ * will reject.
+ */
+const node: Comment = getCommentNodeFromSomewhere();
+
+/**
+ * After this, the comment will become `@property {ІᛖРΟᏒТ('foo').Bar} bar` which
+ * is no longer understandable by TypeScript, but will be allowed by SES. 
+ * 
+ * Note that  this could be avoided entirely by stripping comments during, say,
+ * a bundling phase.
+ */
+evadeCensor(node);
```

Comment:
I prefer `evadeCensor` over `transformSource` in general. Can we use this root for both the stringy and the ASTy APIs?